test/dev/integration.js

```diff
@@ -54,6 +54,25 @@ function testFixture(directory) {
   };
 }
 
+function testFixtureStdio(directory) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      port = ++port;
+      const dev = execa(binaryPath, ['dev', directory, '-p', port]);
+
+      dev.stderr.on('data', async data => {
+        if (data.toString().includes('Ready! Available at')) {
+          resolve({dev, port})
+        }
+      });
+
+      await dev;
```

Comment:
Why is this needed?

Reply:
@AndyBitz Thank for spot it, indeed, it isn't necessary here. removed it.